webui/react/src/hooks/useModal/Experiment/useModalExperimentMove.tsx

```diff
@@ -193,16 +150,16 @@ const useModalExperimentMove = ({ onClose }: Props): ModalHooks => {
             })}
           </SelectFilter>
         </div>
-        {destSettings.workspaceId && destSettings.workspaceId !== 1 && (
+        {workspaceId && workspaceId !== 1 && (
           <div>
             <label className={css.label} htmlFor="project">
               Project
             </label>
-            {destSettings.workspaceId === undefined ? (
+            {workspaceId === 1 ? (
```

Comment:
fix: this condition will never be true because on line 153 above you're only rendering this part if `workspaceId !== 1`. This condition should be changed back to `workspaceId === undefined ? ...`

Reply:
workspaceId will also never be undefined so it seems like this is just dead code.